src/components/menu/MainMenu.tsx

```diff
@@ -78,11 +140,16 @@ export default function Banner(props: { [x: string]: any }) {
 					_focus={{
 						bg: 'transparent'
 					}}
-					mb='10px'>
+					mb='10px'
+					onClick={() => {
+						const editorContent = document.getElementById("editor-main")?.innerText;
```

Comment:
Make it inner html to capture formatings as well? Or maybe we can have two different downloads one formatted one plain text?

Reply:
I can't do that because .txt doesn't capture formatting. If I replaced it with '.innerHTML,' I get the text with <b> and other formatting tags as text. I'll implement the feature in Export as an HTML file to capture the non-plain formatting.

Reply:
So this button is meant to save in plain text? hmmm... Ok.